src/main/java/org/spectra/cluster/cdf/CumulativeDistributionFunctionFactory.java

```diff
@@ -1,11 +1,10 @@
 package org.spectra.cluster.cdf;
 
+import com.google.common.io.LineReader;
 import org.spectra.cluster.similarity.CombinedFisherIntensityTest;
 
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.util.Objects;
+import java.io.InputStream;
+import java.io.InputStreamReader;
```

Comment:
java.nio is really fast compared with common java.io @jgriss. We have used in other implementations and the performance increase is really high. 

Now, I see that is only used for File creation. We need to keep track of this java 10 issue if we want to use for reading the java.nio. For now, it looks fine.